node-package/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Version can be found at: https://chromedriver.storage.googleapis.com/LATEST_RELEASE
+substitutions:
+    _VERS: "2020-11-05"
+
+  steps:
+    - name: "gcr.io/cloud-builders/docker"
+      args:
+        - build
+        - "--build-arg=VERS=${_VERS}"
+        - --tag=gcr.io/datcom-ci/node-package:${_VERS}
```

Comment:
can just call it "node" instead of "node-package" everywhere

Reply:
fixed